test-it/ExpectedResults/TestVI_VariantsAreEqual.vtr

```diff
@@ -0,0 +1,2 @@
+uInt32VariantsAreEqual = false
```

Comment:
Remove this test result and the associated VIA file. They were used as part of developing the feature. You already have better tests.

Reply:
Done.